packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -620,21 +815,71 @@ void main() {
     expect(find.byKey(optionsKey), findsOneWidget);
 
     // Options are just below the field.
-    final Offset optionsOffset = tester.getTopLeft(find.byKey(optionsKey));
-    Offset fieldOffset = tester.getTopLeft(find.byKey(fieldKey));
+    final Offset optionsTopLeft = tester.getTopLeft(find.byKey(optionsKey));
+    final Offset fieldOffset = tester.getTopLeft(find.byKey(fieldKey));
     final Size fieldSize = tester.getSize(find.byKey(fieldKey));
-    expect(optionsOffset.dy, fieldOffset.dy + fieldSize.height);
+    expect(optionsTopLeft.dy, fieldOffset.dy + fieldSize.height);
 
     // Move the field (similar to as if the keyboard opened). The options move
     // to follow the field.
     setState(() {
       alignment = Alignment.topCenter;
     });
     await tester.pump();
-    fieldOffset = tester.getTopLeft(find.byKey(fieldKey));
-    final Offset optionsOffsetOpen = tester.getTopLeft(find.byKey(optionsKey));
-    expect(optionsOffsetOpen.dy, isNot(equals(optionsOffset.dy)));
-    expect(optionsOffsetOpen.dy, fieldOffset.dy + fieldSize.height);
+    final Offset fieldOffsetFrame1 = tester.getTopLeft(find.byKey(fieldKey));
+    final Offset optionsTopLeftOpenFrame1 = tester.getTopLeft(find.byKey(optionsKey));
+    // TODO(justinmc): The options don't move when alignment changes like this :(
```

Comment:
Is this comment still applicable?

Reply:
It looks like this test passes, so the comment can be removed :+1: